docker/docker-compose.yml

```diff
@@ -23,10 +23,10 @@ services:
     image: phenix
     container_name: phenix
     privileged: true
+    pid: host
```

Comment:
What is the difference between `pid: host` and mounting `/proc`? And should we make this change in other docker-compose files for phenix?

Reply:
This let's the phenix container directly access the host's PID tree, which is required for some of the advanced cleanup of image builds if they fail. With just the `/proc` mount, the phenix container doesn't have permissions to view the specific fuser information to be able to target and kill the "hung" processes preventing the unmounting of the chroot environment.